source/plugin/permissions/definitions.rst

```diff
@@ -0,0 +1,181 @@
+===========
+Definitions
+===========
+
+Permission
+==========
+
+A permission is a case-sensitive??? hierarchical string key that is used to determine whether a Subject can do a specific action or not.
+The string is split into separate parts using the dot character.
+Permissions should be structured like this.
+
+.. code-block:: text
+    
+    <PluginID>.<MainGroup>.<Subgroup1>...
+
+Allowed characters are:
+
+* "A" - "Z"
+* "a" - "z"
+* "0" - "9"
+* "_"
+* "-"
+* "."
+
+Examples
+~~~~~~~~
+
```

Comment:
I think it's worth stressing that these are only examples, not requirements.

It is not absolutely necessary that plugins implement "myplugin.commands", nor is it necessary to have ".execute" or ".all".

They're simply suggestions.